recordedit/recordEdit.app.js

```diff
@@ -223,7 +224,11 @@
                                             value = (!isNaN(floatVal) ? floatVal : null);
                                             break;
                                         default:
-                                            value = values[i];
+                                            if (column.isAsset) {                                            
```

Comment:
For an asset column, `column.type.name` is not being set to something specific to assets? Is this because `ermrest` implements these as text fields in the database?